dom.bs

```diff
@@ -1738,32 +1741,23 @@ for a given <a>slot</a> <var>slot</var>, run these steps:</p>
 
 <h5 id=assigning-slotables-and-slots>Assigning slotables and slots</h5>
 
-<p>To <dfn noexport>assign slotables</dfn>, for a <a>slot</a> <var>slot</var> with an optional
-<var>suppress signaling flag</var> (unset unless stated otherwise), run these steps:
+<p>To <dfn noexport>assign slotables</dfn> for a <a>slot</a> <var>slot</var>, run these steps:
 
 <ol>
  <li><p>Let <var>slotables</var> be the result of <a>finding slotables</a> for <var>slot</var>.
 
- <li><p>If <var>suppress signaling flag</var> is unset, and <var>slotables</var> and
- <var>slot</var>'s <a for=slot>assigned nodes</a> are not identical, then run
- <a>signal a slot change</a> for <var>slot</var>.
+ <li><p>If <var>slotables</var> and <var>slot</var>'s <a for=slot>assigned
+ nodes</a> are not identical, then run <a>signal a slot change</a> for <var>slot</var>.
```

Comment:
Yes, that is correct and intentional.
- For assigned nodes' changes, the condition became relaxed.
- For other cases (aka fallback contents), the condition become more tight.